src/js/modules/utilities/Logging.ts

```diff
@@ -9,40 +9,99 @@
  */
 
 import * as winston from "winston";
+import { Container, Logger, LoggerOptions } from "winston";
 
-export function newLogger(
```

Comment:
I would seperate this change into two commits:
1. Modifications to Logging.ts and the core logging framework that you want to provide
2. Changes to other file that modify log lines or change how other modules obtain a reference to a logger

Reply:
If I split it, the first commit crash on tests because I change the way how we create a logger

Reply:
ok